ONSdigital/eq-terraform
# Add support for rsyslogd support on RabbitMQ servers

**What**
This PR adds support for increased security provisioning on the rabbitmq
servers, adding rsyslog support and increased security controls.

**How to test**
1. Make sure that https://github.com/ONSdigital/eq-messaging/pull/7 has been merged
2. Set `rsyslog_server_IP` in `terraform.tfvars` to a correct value for rsyslog
3. Deploy this terraform branch in dev.
4. Check that the correct value is passed into the ansible provisioning of the rabbitmq servers.
5. Check that the Rabbitmq servers have correctly been provisioned by checking the config files on the servers.

**Who can test**

Anyone but @dhilton
